### ISSUE TITLE
Include uniform distribution in calibration parameters

### ISSUE THREAD

**Issue by tadhg-moore** [aemon-j/LakeEnsemblR#48]
An alternative to Latin hypercube sampling would be to uniformly sample within the ranges. This would be particularly useful when there are logarithmic parameters e.g. k_min to ensure that the logarithmic distribution is sampled effectively. 

Useful function for this here: https://rdrr.io/cran/KScorrect/man/dlunif.html

**Comment by robertladwig** [comment]
But are there any parameters that really have a uniform distribution, as I would reckon we don't know the real PDF of any parameter. Wouldn't then uniformly sampling be slow by assuming that every point in the range has the same probability?

**Comment by tadhg-moore** [comment]
I noticed it when I used LHS on a logarithmic parameter and it didn't sample the parameter space  thoroughly esepcially where the parameter was had a better fitness. See the plot where I did LHC on the full space but didn't account for the log but using a uniform distribution (red square) I was able to sample the logarithmic space thoroughly. And the idea of this is to provide information on the parameter space before a more targeted calibration tool would be used.

![image](https://user-images.githubusercontent.com/23136262/72605689-2705a800-3915-11ea-837e-9ee875333a0d.png)
